### PR TITLE
carloshz4/c03-docker01

### PR DESCRIPTION
**Pull request by carloshz4**
# Description

Exercise (e.g. C01-E01): c03-docker01

Student Name: Carlos Zambrano

> Added README.md and Dockerfile inside folder classes/03class/exercises/c03-docker01/carloshz4/

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker01/carloshz4/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Command Execution Output
+- Command to build the image:
+```
+docker build -t da-terraform:c03-docker-e01 .
+```
+
+- Command to run the image and its output.:
+```
+docker run -it da-terraform:c03-docker-e01
```
Comment (alanlima) [inline classes/03class/exercises/c03-docker01/carloshz4/README.md:14]: It is missing the output of this command 😅

Comment (carloshz4) [reply classes/03class/exercises/c03-docker01/carloshz4/README.md:18]: Thanks Alan, you are going reviewing these exercises mate.  I've updated those missing commands.

File: classes/03class/exercises/c03-docker01/carloshz4/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+- Command to build the image:
+```
+docker build -t da-terraform:c03-docker-e01 .
+
+docker images |egrep "REPOSITORY|da-terraform"
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+da-terraform        c03-docker-e01      ec3cbdf038c8        16 hours ago        52.7MB
```
Comment (alanlima) [inline classes/03class/exercises/c03-docker01/carloshz4/README.md:13]: Nice size. Well done 👌

File: classes/03class/exercises/c03-docker01/carloshz4/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Docker 
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+- Command to build the image:
+```
+docker build -t da-terraform:c03-docker-e01 .
+
+docker images |egrep "REPOSITORY|da-terraform"
```
Comment (alanlima) [inline classes/03class/exercises/c03-docker01/carloshz4/README.md:11]: Another way is to use [`--filter` option](https://docs.docker.com/engine/reference/commandline/images/#filtering) like: `docker images --filter reference=da-terraform`

File: classes/03class/exercises/c03-docker01/carloshz4/README.md (hunk 1 above)
Comment (alanlima) [reply classes/03class/exercises/c03-docker01/carloshz4/README.md:18]: You're welcome 😁

**Pull request merged**